test/blackbox/common/BlackboxTestsNetworkConf.cpp

```diff
@@ -290,6 +290,84 @@ TEST_P(NetworkConfig, PubSubInterfaceWhitelistUnicast)
     reader.block_for_all();
 }
 
+// Regression test to check in UDP (v4) that setting the interface whitelist in one of the endpoints (writer),
```

Comment:
NIT
The Redmine Task or Issue # could be added here in the comment for further reference (in the project, search for `Regression test for` to see other examples in the code)